<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExigencesDialog</class>
 <widget class="QDialog" name="ExigencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>UV souhaitée</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="wish"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="addWish">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/new/icons/add.png</normaloff>:/new/icons/add.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWish"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>UV non souhaitée</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="reject"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="addReject">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/new/icons/add.png</normaloff>:/new/icons/add.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableReject"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExigencesDialog</receiver>
   <slot>resolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExigencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addWish</sender>
   <signal>clicked()</signal>
   <receiver>ExigencesDialog</receiver>
   <slot>wishAdded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addReject</sender>
   <signal>clicked()</signal>
   <receiver>ExigencesDialog</receiver>
   <slot>rejectAdded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resolve()</slot>
  <slot>wishAdded()</slot>
  <slot>rejectAdded()</slot>
 </slots>
</ui>
